core/storage/user/gae_models.py

```diff
@@ -2380,14 +2380,13 @@ class UserGroupModel(base_models.BaseModel):
 
     This model stores the user-groups for which feature-flag or
     platform-parameters can be enabled.
-
-    ID for this model is the name of the user-group.
     """
 
     # We use the model id as a key in the Takeout dict.
     ID_IS_USED_AS_TAKEOUT_KEY: Literal[True] = True
 
     # The user ids of the users.
+    name = datastore_services.StringProperty(required=True, indexed=True)
```

Comment:
The comment above is wrong, it does not describe this line,

The name below should also be clarified ("users" is vague).

Reply:
Updated